doc/usage.md

```diff
@@ -0,0 +1,109 @@
+# CodeCompass usage
+
+## Setup database (one time only)
```

Comment:
Yes, I know, but we should write a using example at the bottom of this section like I wrote above.